pandas/core/internals/ops.py

```diff
@@ -102,3 +109,11 @@ def _get_same_shape_values(
         rvals = rvals[0, :]
 
     return lvals, rvals
+
+
+def blockwise_all(left: "BlockManager", right: "BlockManager", op) -> bool:
+    for lvals, rvals, _, _, _, _ in _iter_block_pairs(left, right):
```

Comment:
can you add a doc-string (also using a NamedTuple makes this easier to read here)